yb-voyager/cmd/exportData.go

```diff
@@ -395,6 +399,30 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 	return nil
 }
 
+func reportStreamingProgress() {
+	header := uilive.New()
```

Comment:
`uilive.New()` returns an instance of type `*uilive.Writer`. Should we call the variable as `w` or `writer` instead of `header`?